CHANGELOG.md

```diff
@@ -3,6 +3,14 @@ Changelog for NeoFS Contract
 
 ## [Unreleased]
 
+### Added
+### Changed
+### Updated
+### Fixed
+### Updating from v0.16.0
+
+## [0.16.0] - 2022-10-17
```

Comment:
Should we create link in the file bottom?

Reply:
Fixed